pkg/services/ngalert/state/historian/client.go

```diff
@@ -0,0 +1,67 @@
+package historian
```

Comment:
This utility is copied from weaveworks/common.

It's simple enough, most things sending requests across the project have their own local implementation of this and don't use a shared one. So, keeping it local seemed fine. We can extract it later if we like.

Reply:
Yeah, forgot to vendor a test file here, done